ush/forecast_postdet.sh

```diff
@@ -583,7 +583,7 @@ CICE_out() {
   echo "SUB ${FUNCNAME[0]}: Copying output data for CICE"
 
   # Copy ice_in namelist from DATA to COMOUTice after the forecast is run (and successfull)
```

Comment:
Should probably update this comment
```suggestion
  # Copy ice_in namelist from DATA to COMOUT_CONF after the forecast is run (and successfull)
```